core/src/test/scala/com/nvidia/spark/rapids/tool/qualification/QualificationSuite.scala

```diff
@@ -1154,7 +1154,8 @@ class QualificationSuite extends BaseTestSuite {
         try {
           val lines = inputSource.getLines.toSeq
           // 1 for header, 1 for values
-          assert(lines.size == 6)
+          val expLinesSize = if (ToolUtils.isSpark340OrLater()) 8 else 6
```

Comment:
I am seeing test failure for spark-320. The value seems to be 7 in that case.

Reply:
really? the maven verify check / build should fail then..
I will create a PR to fix the python unit test failure. it is unrelated to our code.

Reply:
Yeah. It failed for spark330. 
```
- test csv output for unsupported operators *** FAILED ***
  Stream("App ID,Unsupported Type,Details,Notes", ""local-1692672769692","Read","JSON","Types not supported - bigint:int"", ""local-1692672769692","Exec","AdaptiveSparkPlan",""", ""local-1692672769692","Exec","Scan json",""", ""local-1692672769692","Exec","Execute InsertIntoHadoopFsRelationCommand json",""", ""local-1692672769692","Exec","LocalTableScan",""", ""local-1692672769692","Write","json",""") had size 7 instead of expected size 6 (QualificationSuite.scala:1158)
```

> I will create a PR to fix the python unit test failure. it is unrelated to our code.
Thank you!